packages/broker/test/integration/multi-account-brokers.test.ts

```diff
@@ -0,0 +1,800 @@
+
+        expect(client3Messages).toEqual([
+            {
+                key: 1
+            },
+            {
+                key: 2
+            },
+            {
+                key: 3
+            },
+        ])
+    })
+
+    it('happy-path: resend last request via websocket', async () => {
+        await Promise.all([
+            client1.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+            client2.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+            client3.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+        ])
+
+        await client1.publish(freshStreamId, {
+            key: 1
+        })
+        await client1.publish(freshStreamId, {
+            key: 2
+        })
+        await client1.publish(freshStreamId, {
+            key: 3
+        })
+        await client1.publish(freshStreamId, {
+            key: 4
+        })
+
+        await wait(3000) // wait for propagation
+
+        const client1Messages: Todo[] = []
+        const client2Messages: Todo[] = []
+        const client3Messages: Todo[] = []
+
+        await Promise.all([
+            client1.resend({
+                stream: freshStreamId,
+                resend: {
+                    last: 2
+                }
+            }, (message) => {
+                client1Messages.push(message)
+            }),
+            client2.resend({
+                stream: freshStreamId,
+                resend: {
+                    last: 2
+                }
+            }, (message) => {
+                client2Messages.push(message)
+            }),
+            client3.resend({
+                stream: freshStreamId,
+                resend: {
+                    last: 2
+                }
+            }, (message) => {
+                client3Messages.push(message)
+            })
+        ])
+
+        await waitForCondition(() => client2Messages.length === 2 && client3Messages.length === 2)
+        await waitForCondition(() => client1Messages.length === 2)
+
+        expect(client1Messages).toEqual([
+            {
+                key: 3
+            },
+            {
+                key: 4
+            },
+        ])
+
+        expect(client2Messages).toEqual([
+            {
+                key: 3
+            },
+            {
+                key: 4
+            },
+        ])
+
+        expect(client3Messages).toEqual([
+            {
+                key: 3
+            },
+            {
+                key: 4
+            },
+        ])
+    })
+
+    it('happy-path: resend from request via websocket', async () => {
+        await Promise.all([
+            client1.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+            client2.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+            client3.subscribe({
+                stream: freshStreamId
+            }, () => {}),
+        ])
```

Comment:
Hm, remind me why it needs to already be subscribed? @harbu I recall something like this before in other tests but don't remember the reason, seems crap. Maybe unnecessary?